lessons/lesson1/example4.jsonnet

```diff
@@ -7,21 +7,29 @@ local webserver = {
       image: 'httpd:2.4',
     },
 
-    deployment: {
-      apiVersion: 'apps/v1',
```

Comment:
The nested deployment object is of importance to write idiomatic jsonnet, it allows for writing multiple objects to exist within a single output.

If we'd like to pipe the jsonnet output to `kubectl`, I think we need to write a wrapper script that extracts the kubernetes objects into a list that is safe to be applied.